packages/auth/e2e/validatePassword.e2e.js

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2016-present Invertase Limited & Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this library except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import { validatePassword } from '../lib/';
+
+describe('auth() -> validatePassword()', function () {
+  it('isValid is false if password is too short', async function () {
+    let status = await validatePassword(firebase.auth(), 'Pa1$');
```

Comment:
`firebase.auth()` --> `getAuth()` - this is modular only, e2e tests should use the modular APIs, otherwise they'll add more/new deprecation warnings to the e2e suite and last I checked it was clean a couple weeks ago - though may need to clean it again best not to add new ones